src/App.tsx

```diff
@@ -50,7 +28,7 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
```

Comment:
The class `is-success` is not a standard Bulma class for the `box` component. Bulma uses `is-success` for color modifiers on elements like buttons, notifications, etc. Consider reviewing the Bulma documentation to ensure correct usage of classes.